src/System Application/App/File System/app.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id":  "c9c54414-80c3-4cc9-98c6-589158882774",
+    "name":  "File System",
+    "publisher":  "Microsoft",
+    "brief":  "Enables user to send emails from Business Central.",
```

Comment:
@IceOnly, could you maybe go through all objects and ensure that the quality is on par with what is expected of a PR which isn't in DRAFT? Simple oversights like copy/paste issues derail code reviews.
Also, for IDs you could e.g. use IDs 9450 and up.

Once you're happy with your PR, please let me know and I'll get started on a proper test and review! 🥳

Reply:
App is renumbered and should be ready to review.
Only the conenctor Apps in the other PR nened an prooper id range and namespace suggestion.